src/View/pagination.html.twig

```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+  <title>Message Board</title>
+</head>
+<body>
+
+{% if lastPage > 1 %}
+
+    {# the number of first and last pages to be displayed #}
+    {% set extremePagesLimit = 3 %}
+
+    {# the number of pages that are displayed around the active page #}
+    {% set nearbyPagesLimit = 2 %}
+
+    <div class="pagination pagination-centered">
+        <ul>
+          {% if currentPage > 1 %}
+            <li><a href="{{ paginationPath }}{{ currentPage-1 }}">&larr;</a></li>
+
+            {% for i in range(1, extremePagesLimit) if ( i < currentPage - nearbyPagesLimit ) %}
+                <li><a href="{{ paginationPath }}{{ i }}">{{ i }}</a></li>
```

Comment:
Ideally `paginationPath` should be using the router and not relying upon someone to pass it in as a variable.

Reply:
we don't use named routes with Silex, would be cool if it was possible, could then create a url helper method for routes. Maybe something to investigate for another task. I'd suggest for the moment, there isn't much benefit in injecting the path from the controller. I'd just hardcode it `<a href="?page={{ i }}">{{ i }}</a>`

Reply:
added another issue for named routes and url twig extension: https://github.com/BiffBangPow/message-board/issues/12